repotracker/repotracker_test.go

```diff
@@ -371,6 +373,13 @@ tasks:
 	build1, err := build.FindOneId(bv.BuildId)
 	assert.NoError(t, err)
 
+	// neither batchtime task nor disabled task should be activated
+	tasks, err := task.Find(task.ByBuildId(build1.Id))
+	assert.Len(t, tasks, 2)
+	for _, tsk := range tasks {
+		assert.False(t, tsk.Activated)
```

Comment:
Can the BV also include a task that should be activated to double-check that it activates a subset of the BV tasks?